enclave/crypto/gcm.c

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <mbedtls/cipher.h>
+
+#include <openenclave/internal/raise.h>
+#include <openenclave/internal/crypto/gcm.h>
+
+oe_result_t oe_aes_gcm_encrypt(
+    const uint8_t* key,
+    size_t keylen,
+    const uint8_t* iv,
+    size_t ivlen,
+    const uint8_t* ad,
+    size_t adlen,
```

Comment:
https://en.wikipedia.org/wiki/Galois/Counter_Mode mentions nothing called "ad" so this is confusing. Either "aad" or spell out whatever this is if it's not the additional authenticated data.  Same in gcm.h.
```suggestion
    const uint8_t* aad,
    size_t aad_size,
```